packages/ui-components/src/numeric-input/numeric-input.tsx

```diff
@@ -165,6 +165,8 @@ export interface NumericInputProps extends InteractiveComponentProps<number> {
     onChange?: (value: number, e?: React.SyntheticEvent<HTMLInputElement>) => void | Promise<void>;
     /** An optional event listener for complete events (enter presses) */
     onComplete?: () => void | Promise<void>;
+    /** An optional event listener for keydown events if needed */
```

Comment:
More of a pedantic one, but the "if needed" at the end doesn't really add much since you already call it an optional, you can just remove it